python/gstaichi/lang/kernel_impl.py

```diff
@@ -721,11 +765,17 @@ def _recursive_set_args(
         for field in needed_arg_fields.values():
             if field._field_type is not _FIELD:
                 continue
+            field_name = field.name
+            field_full_name = py_dataclass_basename + (field_name,)
```

Comment:
I'm wondering whether this is faster than `field_full_name = (*py_dataclass_basename, field_name)`. I would expect the later to be more efficient because one less tuple must be allocated and freed.

Reply:
I'm fairly sure I tested, and this was faster. (I initially tried (*py_dataclass_basename, field_name) ). NOt sure if I documented hti though 🤔